frontend/src/components/object/ObjectFileDetails.vue

```diff
@@ -81,68 +97,111 @@ onBeforeMount( async () => {
   }
 });
 
-watch( [props, getObjects], () => {
-  metadataStore.fetchMetadata({objectId: props.objectId });
-  tagStore.fetchTagging({objectId: props.objectId });
+watch( [props, getObjects], async () => {
+  await metadataStore.fetchMetadata({objectId: props.objectId});
```

Comment:
Just wondering if having two metadata stores is the best pattern.
maybe a single metadata store could have actions that inherit object > version data from the other stores.
I think you already have the object > version relation in the metadataStore in versioStore.ts (findLatestVersionIdByObjectId)
.. so you can get metadata for an object by looking at the latest version, instead of calling a separate action everywhere.
But maybe that pattern has its drawbacks too. I think what you have is fine.

Reply:
Agree with Tim.